### ISSUE TITLE
Render the outline of camping places

### ISSUE THREAD

**Issue by AlertSubject** [organicmaps/organicmaps#3190]
_This issue is a part of my collection of camping related suggestions and improvements, which can be found here: #3189_

Currently there is an icon in the middle but this makes it harder to determine the outlines especially when the camping place is in green areas, as the camping ground is rendered with the same color.
Outlines can be helpful to determine distances and estimate the size of a camping place

Example: there is room for tents at the location of the marker and some tents were even farther away from the road network. 
An outline would make you feel less lost in green and give the user a sense of scale

![signal-2022-08-17-220609](https://user-images.githubusercontent.com/72819077/185233018-b93260f4-87f6-4bbe-8dd9-74935ee699b7.jpeg)

I went ahead and put my MS-Paint skills to the test and added a few gray lines.

![signal-2022-08-17-220609](https://user-images.githubusercontent.com/72819077/185233548-3cc46b4c-70a4-430b-afed-ebb568f214d7.jpeg)

Now this paints a fuller picture of the actual scale (according to me)

**Comment by janusbenissa** [comment]
Is there a border on the OSM map?

**Comment by AlertSubject** [comment]
> Is there a border on the OSM map?

Yes, CartoCSS (the default map) shows a border and different color
https://www.openstreetmap.org/#map=18/54.76705/8.28284
See area here
